themes/default/views/find/Results/xlsx_results.php

```diff
@@ -46,10 +46,7 @@
 	
 	$va_a_to_z = range('A', 'Z');
 	
-	$workbook = new PHPExcel();
-
-	// more accurate (but slower) automatic cell size calculation
-	PHPExcel_Shared_Font::setAutoSizeMethod(PHPExcel_Shared_Font::AUTOSIZE_METHOD_EXACT);
```

Comment:
I had a look at the way AutoSize was handle in new code for file `themes/default/views/administrate/setup/Results/xlsx_results.php`.

Should we add it also use it here?